nbval/plugin.py

```diff
@@ -111,12 +117,23 @@ def pytest_configure(config):
         from .nbdime_reporter import NbdimeReporter
         reporter = NbdimeReporter(config, sys.stdout)
         config.pluginmanager.register(reporter, 'nbdimereporter')
+    if config.option.sanitize_with:
+        warnings.warn("--sanitize-with has been renamed to --nbval-sanitize-with", DeprecationWarning)
+        if config.option.nbval_sanitize_with:
+            raise ValueError("--sanitize-with and --nbval-sanitize-with were both supplied.")
+        config.option.nbval_sanitize_with = config.option.sanitize_with
+    if config.option.current_env:
+        warnings.warn("--current-env has been renamed to --nbval-current-env", DeprecationWarning)
+        if config.option.nbval_current_env:
+            raise ValueError("--current-env and --nbval-current-env were both supplied.")
+        config.option.nbval_current_env = config.option.current_env
     if config.option.nbval or config.option.nbval_lax:
         if config.option.nbval_kernel_name and config.option.current_env:
             raise ValueError("--current-env and --nbval-kernel-name are mutually exclusive.")
 
 
 
+
```

Comment:
```suggestion
```
Fix whitespace change introduced by inexperienced use of github gui merge tool...